<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalc</class>
 <widget class="QWidget" name="DepositCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="replenishment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>replenishment</string>
   </property>
  </widget>
  <widget class="QLabel" name="withdrawal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>withdrawal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="replenishment_in">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="withdrawal_in">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pereodicity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>pereodicity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pereodicity_in">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="capitalization_in">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="capitalization">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>capitalization</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_term_in">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>interest rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_in">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pereodicity_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>deposit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="capitalization_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>deposit term</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_in">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>tax rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_in">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rep_add">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="with_add">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="aquired_interest">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="replenishment_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>tax amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="capitalization_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>aquired interest</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_amount">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="withdrawal_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>deposit amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>619</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calc">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CALCULATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="withdrawal_counter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="replenishment_counter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
